googletest/src/gtest-port.cc

```diff
@@ -58,6 +58,10 @@
 # include <sys/procfs.h>
 #endif  // GTEST_OS_QNX
 
+#if GTEST_OS_AIX
+# include <procinfo.h>
```

Comment:
sys/types.h provides the definition for pid_t.  This is implicitly included in any of the std.* headers.  I will explicitly add it if you want.